### PR TITLE
fix small typo in documentation about CheckButtons.

### PR DESCRIPTION
**Pull request by flothesof**
Just a little typo.

### CODE REVIEW

**Comment by QuLogic** [comment]
There are actually several of this same typo all over the file; do you mind fixing them at the same time?

**Comment by flothesof** [comment]
@QuLogic I added some more corrections as requested. There's still a FIXME left because I couldn't figure what the original author meant.

**Comment by tacaswell** [comment]
:+1: 

Thanks for cleaning up by copy-paste typo :sheep:

File: lib/matplotlib/widgets.py [hunk 1]
```diff
@@ -772,16 +773,16 @@ def disconnect(self, cid):
...
         """
         *targetfig*
-            The figure instance to adjust
+            The figure instance to adjust.
 
         *toolfig*
             The figure instance to embed the subplot tool into. If
-            None, a default figure will be created. If you are using
+            *None*, a default figure will be created. If you are using
             this from the GUI
```
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: I see you fixed the other truncated comment; any ideas what goes here?

File: lib/matplotlib/widgets.py [hunk 2]
```diff
@@ -624,12 +625,12 @@ def disconnect(self, cid):
 
 class RadioButtons(AxesWidget):
     """
-    A GUI neutral radio button
+    A GUI neutral radio button.
 
     For the buttons to remain responsive
-    you much keep a reference to this object.
+    you must keep a reference to this object.
```
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: Might as well unwrap this line too.

File: lib/matplotlib/widgets.py [hunk 3]
```diff
@@ -921,9 +922,9 @@ def funchspace(self, val):
 
 class Cursor(AxesWidget):
     """
-    A horizontal and vertical line span the axes that and move with
-    the pointer.  You can turn off the hline or vline spectively with
-    the attributes
+    A horizontal and vertical line that spans the axes that and moves with
```
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: The second "that" can be removed.

File: lib/matplotlib/widgets.py (hunk 1 above)
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: Oh, I guess this is the one you meant in the original comment, then.

Comment (flothesof) [inline lib/matplotlib/widgets.py]: Yes, exactly. That's what I meant.

File: lib/matplotlib/widgets.py (hunk 3 above)
Comment (flothesof) [inline lib/matplotlib/widgets.py]: Should be ok now.

File: lib/matplotlib/widgets.py [hunk 4]
```diff
@@ -30,9 +30,9 @@ class LockDraw(object):
     desirable under all circumstances, like when the toolbar is in
     zoom-to-rect mode and drawing a rectangle.  The module level "lock"
     allows someone to grab the lock and prevent other widgets from
-    drawing.  Use ``matplotlib.widgets.lock(someobj)`` to pr
+    drawing.  Use ``matplotlib.widgets.lock(someobj)`` to prevent
+    other widgets from drawing while you're interacting with the canvas.
     """
-    # FIXME: This docstring ends abruptly without...
```
Comment (WeatherGod) [inline lib/matplotlib/widgets.py]: heh, I remember adding that comment as a bit of a joke...

File: lib/matplotlib/widgets.py (hunk 2 above)
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: Seem to have missed this one?

**Comment by tacaswell** [comment]
Merging this as it clearly improves things :+1: 

@flothesof If you want to keep fixing up the docs (which is awesome btw), please open a new PR.

I think this is your first mpl contribution; congratulations and thank you!

**Comment by tacaswell** [comment]
backported as aceb626

**Pull request merged**